cosmwasm/contracts/outpost/src/contract.rs

```diff
@@ -0,0 +1,66 @@
+#[cfg(not(feature = "imported"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{DepsMut, Env, MessageInfo, Response};
+use cw2::set_contract_version;
+
+use crate::error::ContractError;
+use crate::execute::execute_swap;
+use crate::msg::{ExecuteMsg, InstantiateMsg, MigrateMsg};
+use crate::state::{Config, CONFIG};
+
+// version info for migration info
+const CONTRACT_NAME: &str = "crates.io:outpost";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+/// Handling contract instantiation
+#[cfg_attr(not(feature = "imported"), entry_point)]
+pub fn instantiate(
```

Comment:
Do we want any rust tests for instantiate? (Is it easy to do?)

Reply:
should be easy enough. Yeah, I can add them